war/src/main/scss/abstracts/theme.scss

```diff
@@ -1,8 +1,34 @@
-:root, .app-theme-picker__picker[data-theme="none"] {
+@use "sass:color";
+
+$colors: (
+  "blue": #007aff,
+  "brown": #a2845e,
+  "cyan": #32ade6,
+  "green": #1ea64b,
+  "indigo": #5856d6,
+  "orange": #fe820a,
+  "pink": #fb0f45,
+  "purple": #af52de,
+  "red": #e6001f,
+  "yellow": #ffb31a,
+);
+
+$semantics: (
+  "error": var(--red),
+  "warning": var(--orange),
+  "success": var(--green),
+  "destructive": var(--red),
+  "build": var(--green),
+);
+
+:root,
+.app-theme-picker__picker[data-theme="none"] {
```

Comment:
The `.app-theme-picker__picker` CSS used to be present in the compiled CSS for `add-page.css`, `simple-page.css`, and `styles.css`. If its removal from `add-page.css` was intentional then please ignore this comment. If it needs to remain in all three then please restore it to `add-page.css`.

Reply:
Good spot! The 'New Item' page imports 'styles.css' anyways so the additional theme import isn't necessary 👍